thread/linux/omrthreadnuma.c

```diff
@@ -546,3 +547,17 @@ dumpNumaInfo() {
 
 }
 #endif
+
+
+uintptr_t
+omrthread_numa_get_current_node(){
+    unsigned node = 0;
+#if OMR_PORT_NUMA_SUPPORT
+    if (0 == syscall(SYS_getcpu, NULL, &node, NULL)) {
+        ++node;
```

Comment:
This feels like there should be a comment here.  Is the numbering schedule 0 based vs 1 based elsewhere?

Reply:
@dnakamura was this comment addressed somewhere else?